extensions/adobe/experience/offer-management/offer-placement.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/offer-placement",
+  "type": "object",
+  "title": "Placement",
+  "description": "Offer Placement",
+  "definitions": {
+    "offer-placement": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "title": "ID",
+          "description": "The unique identifier of the placement"
+        },
+        "xdm:name": {
+          "type": "string",
+          "title": "Name",
+          "description": "Offer placement name"
+        },
+        "xdm:description": {
+          "type": "string",
+          "title": "Description",
+          "description": "Offer placement description"
+        },
+        "xdm:channel": {
+          "type": "string",
+          "title": "Channel",
+          "description": "Offer Channel",
+          "enum": ["email", "mobile", "web"],
+          "meta:enum": {
+            "email": "Email",
+            "mobile": "Mobile",
+            "web": "Web"
+          }
+        },
+        "xdm:contentTypes": {
```

Comment:
What are content types?  Are they the same as MIME or media types?

Reply:
See https://github.com/adobe/xdm/pull/245/commits/24e6d701366014e9724462f1834193de7f7d4d7c